proxy,status,location,real_location,error
http://102.132.36.193:8080,0,za South Africa - Johannesburg,Failed,"ConnectionError: ('Connection aborted.', BadStatusLine('<html>\r\n'))"
http://164.163.42.12:10000,200,ar Argentina - Villa Ángela,America/Argentina/Buenos_Aires,
http://188.72.43.145:47193,0,iq Iraq,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4b1c8b0>, 'Connection to 188.72.43.145 timed out. (connect timeout=5)')))"
http://77.228.182.122:8080,0,es Spain - Mérida,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4b1d3f0>, 'Connection to 77.228.182.122 timed out. (connect timeout=5)')))"
http://103.131.18.58:8080,0,id Indonesia - Bekasi,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4d06bc0>, 'Connection to 103.131.18.58 timed out. (connect timeout=5)')))"
http://119.252.173.26:8080,0,id Indonesia,Failed,"TimeoutError: HTTPSConnectionPool(host='ipinfo.io', port=443): Read timed out. (read timeout=5)"
http://45.70.202.160:999,0,ec Ecuador - Quito,Failed,"TimeoutError: HTTPSConnectionPool(host='ipinfo.io', port=443): Read timed out. (read timeout=5)"
http://101.255.148.241:8080,0,id Indonesia - Bekasi,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4b1f1c0>, 'Connection to 101.255.148.241 timed out. (connect timeout=5)')))"
http://62.92.103.227:8080,0,no Norway - Oslo,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4d069e0>, 'Connection to 62.92.103.227 timed out. (connect timeout=5)')))"
http://8.213.222.247:77,0,th Thailand - Bangkok,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4d07130>, 'Connection to 8.213.222.247 timed out. (connect timeout=5)')))"
http://164.163.1.14:2180,0,br Brazil - Brasília,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4d04550>, 'Connection to 164.163.1.14 timed out. (connect timeout=5)')))"
http://200.95.184.5:999,0,cl Chile - Chillan,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4b58850>, 'Connection to 200.95.184.5 timed out. (connect timeout=5)')))"
http://8.213.222.247:3128,0,th Thailand - Bangkok,Failed,"ConnectionError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:1007)')))"
http://117.54.114.99:80,0,id Indonesia - Jakarta,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4d05d50>, 'Connection to 117.54.114.99 timed out. (connect timeout=5)')))"
http://103.176.97.223:3127,0,id Indonesia - Karawang,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4d048e0>, 'Connection to 103.176.97.223 timed out. (connect timeout=5)')))"
http://109.164.38.201:2306,0,cz Czech Republic - Prague,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4b1ead0>, 'Connection to 109.164.38.201 timed out. (connect timeout=5)')))"
http://117.54.114.97:80,0,id Indonesia - Jakarta,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4b1e2c0>, 'Connection to 117.54.114.97 timed out. (connect timeout=5)')))"
http://41.60.26.210:32650,0,cd DR Congo,Failed,"ProxyError: HTTPSConnectionPool(host='ipinfo.io', port=443): Max retries exceeded with url: /json (Caused by ProxyError('Unable to connect to proxy', ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ed7f4b1f1f0>, 'Connection to 41.60.26.210 timed out. (connect timeout=5)')))"
